content/manager/implement-multi-tenancy.md

```diff
@@ -0,0 +1,105 @@
+
+The default tenant that is created during the Cloudify Manager installation/bootstrapping and its `administrator` cannot be deleted.
+
+Using the API, an Administrator user can delete a resource on a tenant other than the one which they are currently managing.
+
+
+###Integrating Multi-Tenancy with LDAP/AD###
+
+To connect Cloudify Manager with LDAP/AD, you must know the the URL of the service and have sufficient credentials to perform searches and so on. 
+
+You configure Cloudify with the LDAP configuration during the bootstrap process, in the `manager-input` section. You can also use the API for the configuration, if the Cloudify Manager is clean.
+
+####How Cloudify Manager Works with the LDAP/AD Service####
+
+When a user logs in to Cloudify Manager, their credentials are passed to the LDAP/AD service for authentication. By default, all users in the LDAP/AD service are authenticated to Cloudify Manager, however only users who have specific permissions for a tenant can access it. When a user logs into Cloudify Manager, the service authenticates the user and returns a list of any groups to which the user belongs. If a Cloudify Manager instance has specified a group name for a tenant that has the same name as a group specified in LDAP/AD, all users in that group can access the tenant (unless an individual user is specifically suspended.) In addition, individual users can be given access to a tenant. The following graphic indicates how Cloudify Manager interacts with an LDAP/AD service. 
+
+![User/LDAP relationship]({{< img "manager/multi-tenancy-ldap-relationship.png" >}})
+
+####Managing Users in the Multi-Tenancy Environment####
+
+There are a number of actions related to user access to tenants that an **`administrator`** user can perform. 
+
+{{% gsNote title="Note" %}}
+Although Cloudify is for the most-part case-sensitive, user and group names/DNS are case-insensitive, to align with LDAP protocols.
+{{% /gsNote %}}
+
+#####Tenant-Related Commands#####
+
+To run these tenant-specific commands, use `cfy tenant`.
+
+- `create` enables you to create a tenant
+- `delete` enables you to delete a tenant
+- `get` enables you to view information about the tenant, including its users
+- `list` provides a list of all tenants in this instance of Cloudify Manager
+
+#####User-Related Commands#####
+
+To run these commands, use `cfy users`.
+
+- `add-user` enables you to add an individual user to a tenant
+- `add-user-group` enables you to add a group of users to a tenant
+- `remove-user` enables you to remove a specific user from a group
+- `remove-user-group` enables you to remove a user group from a tenant
+- `set-password` enables you to set the password for a user (in the case of a non-LDAP/AD setup)
+- `set-role` enables you to set the role for a user. Possible roles are `administrator`, `user`, and `suspended`. An `administrator` user can perform all the commands listed here. A suspended user cannot access a tenant.
```

Comment:
We should also mention that the default role assigned to a user is "User", and specify what is the different between user role and admin role (Admin - Can access all of the Manager’s Tenants, 
User - Have access permissions for all public resources in a specific tenant)
***Suspended -A user that exists in the system but has no permission whatsoever but we are changing that soon so it won't be a role but an "inactive" flag. **